src/protocols/Protocols.h

```diff
@@ -39,8 +39,7 @@ enum CHIPProtocolId
     //
     // NOTE: Do not attempt to allocate these values yourself.
 
-    kProtocol_Protocol_Common     = (kChipVendor_Common << 16) | 0x0000, // Common Protocol
-    kProtocol_SecureChannel       = (kChipVendor_Common << 16) | 0x0001, // Secure Channel Protocol
+    kProtocol_SecureChannel       = (kChipVendor_Common << 16) | 0x0000, // Secure Channel Protocol
     kProtocol_Echo                = (kChipVendor_Common << 16) | 0x0002, // Echo Protocol
```

Comment:
We're going to fix the rest of these protocol numbers too, to match the spec.  Please file a followup issue?

Reply:
filed https://github.com/project-chip/connectedhomeip/issues/5441